js/console_test.go

```diff
@@ -371,40 +371,39 @@ func TestFileConsole(t *testing.T) {
 							hook := logtest.NewLocal(logger)
 
 							err = vu.RunOnce()
-							assert.NoError(t, err)
+							require.NoError(t, err)
 
 							// Test if the file was created.
 							_, err = os.Stat(logFilename)
-							assert.NoError(t, err)
+							require.NoError(t, err)
 
 							entry := hook.LastEntry()
-							if assert.NotNil(t, entry, "nothing logged") {
-								assert.Equal(t, level, entry.Level)
-								assert.Equal(t, result.Message, entry.Message)
+							require.NotNil(t, entry, "nothing logged")
+							assert.Equal(t, level, entry.Level)
+							assert.Equal(t, result.Message, entry.Message)
 
-								data := result.Data
-								if data == nil {
-									data = make(logrus.Fields)
-								}
-								assert.Equal(t, data, entry.Data)
+							data := result.Data
+							if data == nil {
+								data = make(logrus.Fields)
+							}
+							require.Equal(t, data, entry.Data)
 
-								// Test if what we logged to the hook is the same as what we logged
-								// to the file.
-								entryStr, err := entry.String()
-								assert.NoError(t, err)
+							// Test if what we logged to the hook is the same as what we logged
+							// to the file.
+							entryStr, err := entry.String()
+							require.NoError(t, err)
 
-								f, err := os.Open(logFilename)
-								assert.NoError(t, err)
+							f, err = os.Open(logFilename) //nolint:gosec
```

Comment:
This is weird :confused: `getSimpleRunner()` uses `afero.NewMemMapFs()` by default, so why (and how) are we polluting the real filesystem here? :confused:

Reply:
We don't use afero in the console code at all - I guess this is a thing we can open a issue about :thinking: 

https://github.com/grafana/k6/blob/aa064148a11fa70defc300b513a5a20f7bf21868/js/console.go#L44

Reply:
https://github.com/grafana/k6/issues/2565